students/ZachCooper/session03/mailroom.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python3
+
+
+
+# Create donors
+donors = [("Fred Jones", [100, 200, 300]),
+		("Amy Schumer", [2000, 4000, 1000]),
+        ("Bill Gates", [5000, 10000, 30000]),
+        ("Chris Fermstad", [300, 1000, 750]),
+        ("Ryan Simmers", [100000, 50000, 100])]
+
+# Command Script that gives user options
+def main():
+	print("Welcome to MailRoom!")
+    answer = ""
+    while answer != "q":
+        print("Please select from the following")
+        print("Quit: 'q', \n"
+            "Thank You: 't'\n"
+            "Report: 'r'")
+        answer = input("=>")
+        answer = answer.strip()
+        answer = answer[:1].lower()
+        if answer == 't':
+            thank_you(answer)
+        elif answer == 'r':
+            create_report()
+        elif answer == 'q':
+            print("Goodbye!!!")
+            break
+        else:
+            print("Invalid choice. Please select 'T', 'Q', or 'R' ")
+
+# Function that sends new_donor or returning donor who donates more a Thank You email
+def send_email(name, amount):
+    """This function will compose the thank you email"""
+    print("Thank you, {}, for your donation of ${}!"
+    "Your contributions are always welcome and look forward to future donations.".format(name, amount))
+
+# Thank you function either prints current donors, calls function for current donor to re-donate, or prompts new donor function
+def thank_you(answer):
+    if answer == 't':
+		name = input("Please enter a donors full name or type 'list' to view list of donor names?")
+        if name == 'list':
+            print(donors)
+        elif name == donors:
```

Comment:
name is never going to equal the entire donor list!

you need a "find_donor" function, or something like that, that looks through the list for a particular donor name.